apiserver/src/api.rs

```diff
@@ -161,6 +173,7 @@ pub async fn run_server<T: 'static + BottlerocketNodeClient>(
                 ))
                 .exclude(APISERVER_HEALTH_CHECK_ROUTE),
             )
+            .wrap(request_metrics.clone())
```

Comment:
Is it possible to prevent this module from gathering or exposing metrics on the shallow health check `/ping` endpoint?

Reply:
As per discussion, the existing actix-web-opentelemetry library doesn't provide an option to filter the requests. However, the generated metrics have different key:value pair for route, so it should be easy for the user to filter on the prometheus end.